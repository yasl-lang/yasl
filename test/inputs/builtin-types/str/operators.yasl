echo 'str1' < 'str2'
echo 'str1' > 'str2'
echo 'str1' <= 'str2'
echo 'str1' >= 'str2'

echo 'str1' < 'str12'
echo 'str1' <= 'str12'
echo 'str1' >= 'str12'
echo 'str1' > 'str12'

echo 'str1' > 'str1'
echo 'str1' < 'str1'
echo 'str1' >= 'str1'
echo 'str1' <= 'str1'

echo 'str1' == 'str2'
echo 'str1' == 'str12'
echo 'str1' == 'str1'
