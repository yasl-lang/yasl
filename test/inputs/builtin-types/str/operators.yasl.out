true
false
true
false
true
true
false
false
false
false
true
true
false
false
true
